pkg/sdn/plugin/subnets.go

```diff
@@ -155,6 +165,37 @@ func (master *OsdnMaster) watchNodes() {
 	}
 }
 
+// Because openshift-sdn uses an overlay and doesn't need GCE Routes,
+// we need to clear the kubelet-added NetworkUnavailable condition
+// that kubelet adds to initial node status when using GCE.
+// TODO: make upstream kubelet more flexible with overlays and GCE
+func (node *OsdnNode) clearInitialNodeNetworkingCondition() error {
+	knode, err := node.registry.GetNode(node.hostName)
+	if err != nil {
+		return err
+	}
+
+	for i := 0; i < updateNodeStatusMaxRetries; i++ {
```

Comment:
As before, this follows vendor/k8s.io/kubernetes/pkg/controller/route/routecontroller.go.  I'd imagine we *do* need to retry in other places, just that we haven't hit that issue yet because not many things are touching HostSubnet or ClusterNetwork objects.  But if we start touching more common objects, there's a bigger chance something else will have updated them before we get there, and our resource version will be out of date, and the update will fail.